### DISCUSSION TITLE
Welcome to truth-generator Discussions!

### DISCUSSION THREAD

**Discussion by astubbs** [astubbs/truth-generator#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by astubbs** [comment]
hi @dmitry-timofeev! Welcome! You're the first starer! I'm curious on your thoughts and ideas for the library?

**Reply by dmitry-timofeev** [reply]
Hi @astubbs , I haven't yet tried it, but thanks a lot for implementing and sharing it!
